master/buildbot/test/integration/sample_data.sql

```diff
@@ -0,0 +1,548 @@
+PRAGMA foreign_keys=OFF;
```

Comment:
I think these are valid points, but I'm not fully convinced :) Let's do this: I will convert the DB to insertTestData and I will see for myself whether my reservation is correct. If you're right then you will not need to spend time just because I'm opposed.

Regarding the code completion of fakedb rows I think it makes sense to add `__init__` with expected arguments and just pass it up to `super()`. This would effectively replace the `defaults` dict which is confusing as it requires the reader to go to `Row` in order to see how `defaults` is used.

Reply:
Created this PR to solve the code completion problem: https://github.com/buildbot/buildbot/pull/6208

Reply:
@tardyp I've converted the sqlite dump to `insertTestData` style, took about a hour. I think this result is more approachable, reviewable and concise.

I think in the future we could refactor the test setup code to look more like this:
```
builder1 = master1.add_builder(name='runtests1', tags=['tag1', 'tag2', 'tag3'])
builder2 = master1.add_builder(name='runtests2', tags=['tag2', 'tag4'])
bs1 = master1.add_buildset(...)
build1, br1 = builder1.add_build(bs1, ...)
```

That would remove a lot of the test boilerplate.